packages/create-amplify/src/amplify_project_creator.ts

```diff
@@ -71,8 +71,12 @@ export class AmplifyProjectCreator {
 
     logger.log(
       `Welcome to AWS Amplify! 
-Run \`npx amplify help\` for a list of available commands. 
-Get started by running \`${cdCommand}npx amplify sandbox\`.`
+Run \`${
```

Comment:
Rather than have the PackageMangerController expose it's binary name (which could encourage misuse, we can have a method like `packageManagerController.getWelcomeMessage()`
To prevent too much duplication, we can define the following function in a common location that can be used by all the PackageManagerControllers:
```ts
export const generateWelcomeMessage = (executableName: string) => `Run ${executableName} amplify help for a list of available commands. Get started by running ${executableName} amplify sandbox`
```
The implementation for `getWelcomeMessage` on the NpmPackageManagerController would then look like:
```ts
import { generateWelcomeMessage } from './generate_welcome_message.js';

export class NpmPackageManagerController implements PackageManagerController {
  getWelcomeMessage = () => {
    return generateWelcomeMessage('npx');
  }
}
```